docs/mkdocs/docs/sdk-docs.md

```diff
@@ -4,19 +4,23 @@ This is the documentation for Whitebox's SDK. For an interactive experience, you
 
 ## Models
 
-**_create_model_**_(name, type, features, prediction, probability, labels, description="")_
+**_create_model_**_(name, type, features, prediction, probability, labels=None, description="")_
```

Comment:
You're adding this by hand at the moment, right?
There is an automated way to create this. Not sure how it would work with the styles though. 
Maybe do some research in the future.

Reply:
I don't really get what you mean...

Reply:
Im saying that you can automate this.
No worries. Its fine ;)